app/domain/operations/notices/ivl_enr_notice_trigger.rb

```diff
@@ -83,11 +84,11 @@ def build_family_member_hash(enrollment)
               person_health: { is_tobacco_user: person.is_tobacco_user },
               is_active: person.is_active,
               is_disabled: person.is_disabled,
-              consumer_role: build_consumer_role(person.consumer_role),
               addresses: build_addresses(person, fm.is_primary_applicant)
             }
           }
           member_hash[:person].merge!(verification_types: update_and_build_verification_types(person)) if outstanding_verification_types.present?
+          member_hash[:person].merge!(consumer_role_ref || resident_role_ref)
```

Comment:
There should only ever be either consumer or resident role

Reply:
Confirmed this is the case, but we need to use these methods as our source of truth for which role is active:
```
 def is_consumer_role_active?
    self.active_individual_market_role == "consumer" ? true : false
  end

  def is_resident_role_active?
     self.active_individual_market_role == "resident" ? true : false
  end
```